README.md

```diff
@@ -153,9 +153,90 @@ This library doesn't handle this for you, but here are the 2 main options:
 This is the easiest method: by rendering your form inside a live component,
 it will automatically re-render when the form changes.
 
-### B) Write custom JavaScript
+### B) Use [Symfony UX Turbo](https://symfony.com/bundles/ux-turbo/current/index.html#decomposing-complex-pages-with-turbo-frames)
+
+By using turbo frames you can have a dynamic form running quickly without any JavaScript.
+
+Simply add a `<turbo-frame>` around your form:
+
+```twig
+<turbo-frame id="rating-form">
+{{ form_start(form) }}
+    {{ form_row(form.rating) }}
+
+    {% if form.badRatingNotes is defined %}
+        {{ form_row(form.badRatingNotes) }}
+    {% endif %}
+    {# We also add a real submit button #}
+    {{ form_row(form.submit) }}
+{{ form_end(form) }}
```

Comment:
Let's indent everything in `turbo-frame` tag with a tab please :)

Reply:
Btw, I don't see any value of showing the expanded form rendering here. wha about to simplify the whole example to just `{{ form(form) }}`? It would already mean that you just need to render your form in that spot. If you have custom rending with `form_row()` etc - that's up to you. Because right now it seems like we're focusing on rendering something important between that `form_start`/`form_end` to make it work, but that's not true if I understand it correctly

Reply:
Right, the only thing important is to have a proper `submit` button, but as it is specified in the `FormType`, there is no need to have it fully expanded here.

I'll simplify it!